app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -0,0 +1,172 @@
+  mapView.addOnFirstLayoutListener { _, _, _, _, _ ->
+    // Limit the vertical scrollable area to avoid the user scrolling too far from the hike
+    mapView.setScrollableAreaLimitLatitude(
+        min(MapScreen.MAP_MAX_LATITUDE, mapView.boundingBox.latNorth),
+        max(MapScreen.MAP_MIN_LATITUDE, mapView.boundingBox.latSouth),
+        HikeDetailScreen.MAP_BOUNDS_MARGIN)
+    if (route.bounds.maxLon < HikeDetailScreen.MAP_MAX_LONGITUDE ||
+        route.bounds.minLon > HikeDetailScreen.MAP_MIN_LONGITUDE) {
+      mapView.setScrollableAreaLimitLongitude(
+          max(HikeDetailScreen.MAP_MIN_LONGITUDE, mapView.boundingBox.lonWest),
+          min(HikeDetailScreen.MAP_MAX_LONGITUDE, mapView.boundingBox.lonEast),
+          HikeDetailScreen.MAP_BOUNDS_MARGIN)
+    }
+  }
+
+  val hikeLineColor = route.getColor()
+  MapUtils.showHikeOnMap(mapView = mapView, hike = route, color = hikeLineColor, onLineClick = {})
+
+  val errorMessageIdState = profileViewModel.errorMessageId.collectAsState()
+  val profileState = profileViewModel.profile.collectAsState()
+
+  AsyncStateHandler(
+      errorMessageIdState = errorMessageIdState,
+      actionContentDescriptionStringId = R.string.go_back,
+      actionOnErrorAction = { navigationActions.navigateTo(Route.MAP) },
+      valueState = profileState,
+  ) { _ ->
+    Box(modifier = Modifier.fillMaxSize().testTag(Screen.HIKE_DETAILS)) {
+      // Map
+      AndroidView(
+          factory = { mapView },
+          modifier =
+              Modifier.fillMaxWidth()
+                  .padding(bottom = 300.dp) // Reserve space for the scaffold at the bottom
+                  .testTag(TEST_TAG_RUN_HIKE_SCREEN_MAP))
+      // Back Button at the top of the screen
+      BackButton(
+          navigationActions = navigationActions,
+          modifier =
+              Modifier.padding(top = 40.dp, start = 16.dp, end = 16.dp)
+                  .testTag(TEST_TAG_RUN_HIKE_SCREEN_BACK_BUTTON),
+          onClick = { navigationActions.goBack() })
+      // Zoom buttons at the bottom right of the screen
+      ZoomMapButton(
+          onZoomIn = { mapView.controller.zoomIn() },
+          onZoomOut = { mapView.controller.zoomOut() },
+          modifier =
+              Modifier.align(Alignment.BottomEnd)
+                  .padding(bottom = MapScreen.BOTTOM_SHEET_SCAFFOLD_MID_HEIGHT + 8.dp)
+                  .testTag(TEST_TAG_RUN_HIKE_SCREEN_ZOOM_BUTTONS))
+
+      RunHikeBottomSheet()
+    }
+  }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun RunHikeBottomSheet() {
+  val scaffoldState = rememberBottomSheetScaffoldState()
+
+  BottomSheetScaffold(
+      scaffoldState = scaffoldState,
+      sheetContainerColor = MaterialTheme.colorScheme.surface,
+      sheetPeekHeight = MapScreen.BOTTOM_SHEET_SCAFFOLD_MID_HEIGHT,
+      modifier = Modifier.testTag(TEST_TAG_RUN_HIKE_SCREEN_BOTTOM_SHEET),
+      sheetContent = { Text("Empty Bottom Sheet") }) {}
```

Comment:
Nitpick: I think you should remove the Text here, since I don't see any use for it.